plugin/externalPluginEntry.go

```diff
@@ -274,4 +274,19 @@ func (s *stdoutStreamer) Read(p []byte) (int, error) {
 
 func (s *stdoutStreamer) Close() error {
 	return s.cmd.Wait()
+}
+
+func newStdoutDecodeErr(ctx context.Context, decodedThing string, reason error, stdout []byte, example string) error {
+	logMsg := fmt.Sprintf(
+		"could not decode %v from stdout\nreceived:\n%v\nexpected something like:\n%v",
+		decodedThing,
+		strings.TrimSpace(string(stdout)),
+		example,
+	)
+	if ctx == nil {
```

Comment:
When will context be nil? And why are you handling this rather than letting `activity` handle it?

Reply:
Ah, that's clear in the last commit.

Reply:
That's removed in the last commit -- was a mistake